project/AppDependencies.scala

```diff
@@ -38,8 +40,8 @@ private object AppDependencies {
       override lazy val test = Seq(
         "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % scope,
```

Comment:
This can be increase to version 6.0.0
https://mvnrepository.com/artifact/org.scalatestplus.play/scalatestplus-play

Reply:
That's a pre-release version. We don't usually use them because they might not be stable and other dependencies are  likely to be not compatible yet

Reply:
ok